posts/navarroaxel/06.how-to-use-databases-in-the-Docker-world.md

```diff
@@ -0,0 +1,182 @@
+---
+title: How to use databases in the Docker world
+published: true
+description: Run DB engines locally on our machine without installing them by using Docker containers.
+tags: 'docker, database, mongodb, postgres'
+cover_image: ./assets/sign-your-code-cover.png
+id: 1059901
+---
+
+We can run DB engines locally on our machine without installing them by using Docker containers.
+
+💡 Remember that the container's data is removed with the container itself, so, we're going to store the DB data into a specific Docker volume (`--volume`) to keep it in time.
```

Comment:
maybe add a link to the volumes doc here? 
https://docs.docker.com/storage/volumes/

Reply:
```suggestion
💡 Remember that the container's data is removed with the container itself, so, we're going to store the DB data into a specific [Docker volume](https://docs.docker.com/storage/volumes/) (`--volume`) to keep it in time.
```